js/meetup/App.js

```diff
@@ -0,0 +1,190 @@
+import React, { Component } from 'react';
+import Avatar from 'react-avatar';
+import meetups from './data';
+
+const twitter_url = 'https://twitter.com';
+const github_url = 'https://github.com';
+
+function TwitterIcon(props) {
+  if (props.twitter_username) {
+    const twitter_profile = `${twitter_url}/${props.twitter_username}`;
+    return <a href={twitter_profile} target="_blank"><i className="fab fa-twitter speaker-twitter"></i></a>
+  } else {
+    return false;
+  }
+}
+
+function GithubIcon(props) {
+  if (props.github_username) {
+    const github_profile = `${github_url}/${props.github_username}`;
+    return <a href={github_profile} target="_blank"><i className="fab fa-github speaker-github"></i></a>;
+  } else {
+    return false;
+  }
+}
+
+function SocialLinks(props) {
+  if (props.session) {
+    return (
+      <p className="speaker-name speaker-social">
+        <br/>
+        <TwitterIcon twitter_username={props.session.twitter_username}/> &nbsp;&nbsp;
+        <GithubIcon github_username={props.session.github_username}/>
+      </p>);
+  }
+}
+
+function SlideAndVideo(props) {
+  const slide_link =  props.session.slide_link;
+  const video_link =  props.session.video_link;
+  const demo_link =  props.session.demo_link;
+  if (props.session && (slide_link || video_link || demo_link)) {
+    return (<div className="slide-and-video">
+      {slide_link && <a href={slide_link} target="_blank"><i className="fas fa-sliders-h"></i>  Slide</a>} &nbsp;&nbsp;
+      {video_link && <a href={video_link} target="_blank"><i className="fab fa-youtube"></i>  Video</a>} &nbsp;&nbsp;
+      {demo_link && <a href={demo_link} target="_blank"><i className="fas fa-laptop"></i>  Demo</a>}
+    </div>);
+  } else {
+    return false;
+  }
+}
+
+class App extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state ={
+      selectedEpisode: this.latestEpisode(),
+      firstEpisode: 0,
+      messageForNotFound: '',
+    };
+
+    this.onStepChange = this.onStepChange.bind(this);
+    this.latestEpisode = this.latestEpisode.bind(this);
```

Comment:
Minor refactor, define function with arrow head `onStepChange  = () => { ... }` to get rid of these ugly binds.

Reply:
oh! Yeah, that would be better. I will remove those ugly binds :)